src/glue/workflow.ts

```diff
@@ -20,14 +20,26 @@ export interface TriggerOptions extends ResourceProps {
  * Configuration for the Glue Workflow resource.
  */
 export interface WorkflowProps extends ResourceProps {
+  /**
+   * A description of the Workflow
+   */
   readonly description?: string;
+  /**
+   * A name of the Workflow
+   */
   readonly name?: string;
 }
 
 export class Workflow extends Resource {
   // Input properties
-  public readonly description?: string;
-  public readonly name?: string;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now